next-netfilx/src/components/LandingPage/LandingButton.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import styled from "styled-components";
+import Lottie from "react-lottie-player";
+import netflixLanding from "../../app/netflixLanding.json";
+import { useRouter } from "next/navigation";
+import { useEffect, useRef, useState } from "react";
+import { motion } from "framer-motion";
+
+const LandingButton = () => {
+  const router = useRouter();
+  const [isClicked, setIsClicked] = useState(false);
+  const audioRef = useRef<HTMLAudioElement | null>();
+
+  const handleAnimationComplete = () => {
+    router.push("/main");
+  };
+
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      audioRef.current = new Audio("/NetflixIntroSound.mp3");
+    }
+  }, []);
+
+  useEffect(() => {
+    if (isClicked && audioRef.current) {
+      audioRef.current.play();
+    }
+  }, [isClicked]);
+
+  const onIntroClick = () => {
+    setIsClicked(true);
+  };
+
+  return isClicked ? (
+    <Container>
+      <LogoLottie
+        loop={false}
+        animationData={netflixLanding}
+        play
+        speed={1.25}
+        onComplete={handleAnimationComplete}
+      />
+    </Container>
+  ) : (
+    <Container>
+      <IntroButton
+        whileHover={{ scale: 1.5 }}
+        onClick={onIntroClick}
+      ></IntroButton>
+    </Container>
+  );
```

Comment:
```suggestion
 return  (
    <Container>
      {isClicked? <LogoLottie
        loop={false}
        animationData={netflixLanding}
        play
        speed={1.25}
        onComplete={handleAnimationComplete}
      /> : <IntroButton
      whileHover={{ scale: 1.5 }}
      onClick={onIntroClick}
    ></IntroButton>}
    </Container>
  ) 
```

아예 이런 식으로 container 컴포넌트는 공통적이니까 내부에서 isClicked 상태를 기반으로 조건부 렌더링해줘도 좋을 것 같아요